Library/Homebrew/test/utils_spec.rb

```diff
@@ -0,0 +1,340 @@
+require "utils"
+require "tempfile"
+require "utils/shell"
+
+RSpec::Matchers.alias_matcher :have_failed, :be_failed
```

Comment:
I think this should go somewhere global, as it's likely to be useful in more places than this (and it's modifying the global scope anyway which is not clear from its present location).